packages/cell/README.md

```diff
@@ -56,13 +56,13 @@ API
 
 Parameter     | Type            | Description
 --------------|-----------------|------------------------------
-`obj.entity`  | `BemEntityName` | Representation of entity name
+`obj.entity`  | `BemEntityName` | Representation of [BEM entity name](https://github.com/bem/bem-sdk/tree/master/packages/entity-name)
 `obj.tech`    | `string`        | Tech of cell
 `obj.layer`   | `string`        | Layer of cell
 
 ### entity
 
-Returns the name of entity.
+Returns the [BEM entity name](https://github.com/bem/bem-sdk/tree/master/packages/entity-name) of this cell.
```

Comment:
Guess we should move this link to the references list and reuse here. Like that:
```md
…
Returns the [BEM entity name][entity-name] of this cell.
…


[entity-name]: https://github.com/bem/bem-sdk/tree/master/packages/entity-name
```

Reply:
Great idea!